src/modules/flight_mode_manager/tasks/Orbit/FlightTaskOrbit.cpp

```diff
@@ -90,7 +90,15 @@ bool FlightTaskOrbit::applyCommandParameters(const vehicle_command_s &command, b
 
 	// commanded heading behaviour
 	if (PX4_ISFINITE(command.param3)) {
-		_yaw_behaviour = command.param3;
+		if (command.param3 - 5.0f < FLT_EPSILON) {
```

Comment:
To be a bit more explicit you could do as in here: https://github.com/PX4/PX4-Autopilot/blob/36d89df0a722e2fca3f23e4d0ddd9da9fb5b7ecc/src/modules/flight_mode_manager/FlightModeManager.cpp#L318
And define the `ORBIT_YAW_BEHAVIOUR_UNCHANGED` in [vehicle_command](https://github.com/PX4/PX4-Autopilot/blob/36d89df0a722e2fca3f23e4d0ddd9da9fb5b7ecc/msg/VehicleCommand.msg#L155-L159)